frontend/src/types.ts

```diff
@@ -635,6 +635,27 @@ export const ProgressionStepTitles = {
   [ProgressionStep.SERVER_STARTED]: 'Server started',
 };
 
+export const AssociatedSteps = {
```

Comment:
Make sure we type this variable, our new array variable, and if you could also do the `ProgressionStepTitles` one as well.

`Record<ProgressionStep, ...whatever the type is...>` -- that way when we add a new enum value, they can keep us honest to filling out things. If you have any questions about this, lemme know.